scripts/local_testnet/vars.env

```diff
@@ -18,7 +18,10 @@ GENESIS_VALIDATOR_COUNT=80
 # Number of validator client instances that you intend to run
 NODE_COUNT=4
 
-GENESIS_DELAY=180
+# Number of seconds to delay to start genesis block.
+# If started by a script this can be 0, if starting by hand
+# use something like 180.
+GENESIS_DELAY=0
```

Comment:
I think this should be >0 by default (60secs maybe?) as we usually also want to check genesis as well when running local testnets.

Reply:
I'm happy with genesis delay 0, tweaking vars.env is always an option.